cli/command/image/build.go

```diff
@@ -504,7 +504,7 @@ type resolvedTag struct {
 // "FROM <image>" instructions to a digest reference. `translator` is a
 // function that takes a repository name and tag reference and returns a
 // trusted digest reference.
-func rewriteDockerfileFrom(ctx context.Context, dockerfile io.Reader, translator translatorFunc, istrusted bool) (newDockerfile []byte, resolvedTags []*resolvedTag, err error) {
+func rewriteDockerfileFrom(ctx context.Context, dockerfile io.Reader, translator translatorFunc) (newDockerfile []byte, resolvedTags []*resolvedTag, err error) {
```

Comment:
This should document that it always sets trust enabled, so that it doesn't get accidentally used for something else.

Reply:
@dnephin basically it just resolves images to their digest inside a dockerfile, not change "trust" to enabled correct?

Reply:
Right, it doesn't change the global setting, but it does assume that the global setting is enabled.